app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -6,8 +6,8 @@ class Api::Registry::RegistryProxiesController < Api::V2::ApiController
     before_action :confirm_push_settings, only: [:start_upload_blob, :upload_blob, :finish_upload_blob,
                                                  :chunk_upload_blob, :push_manifest]
     skip_before_action :authorize
-    before_action :optional_authorize, only: [:token]
-    before_action :registry_authorize, except: [:token, :v1_search]
+    before_action :optional_authorize, only: [:token, :catalog]
+    before_action :registry_authorize, except: [:token, :v1_search, :catalog]
```

Comment:
while we are here, the '/v2/' endpoint seems to require auth which isn't really intentional, the clients seem to ignore the 401.  Can you add :ping to this except: []   to fix that?

Reply:
actually.... docker hub does this too, so maybe that excepted, ignore for now i guess

Reply:
I wasn't sure why docker client did this too, which was confusing. Based on the podman discussions it seems as if the design goal was to match docker client as closely as possible. Maybe they'll fix this eventually?